lib/rules/spaced-comment.js

```diff
@@ -281,26 +328,36 @@ module.exports = {
 
             // Checks.
             if (requireSpace) {
-                if (!rule.regex.test(node.value)) {
+                if (!rule.beginRegex.test(node.value)) {
```

Comment:
One more nit - `beginMatched` and `endMatched` might be clearer if they were renamed to something a little more literal like `beginningSpace` and `endSpace`. It would then read like:
```js
if (requireSpace) { 
  if (!beginningSpace) {
    // ...
  }
}
```